logbucket/logbucket.go

```diff
@@ -102,14 +103,26 @@ func NewELBDownloader(sess *session.Session, bucketName, bucketPrefix, lbName st
 		BucketName: bucketName,
 		Prefix:     bucketPrefix,
 		LBName:     lbName,
+		LBType:     lbType,
 	}
 }
 
 // pass in time.Now().UTC()
 func (d *ELBDownloader) ObjectPrefix(day time.Time) string {
+	
+	var urlModifier string
+	
+	if d.LBType == "alb" {
```

Comment:
These strings (`alb`, `elb`) would make lovely exported `const` definitions in this file that the `cli/` files could also use (that way, it's transformed to a compile time check instead of runtime).